From 3548421e09d714c23eb74ed7302221df8274d356 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Wed, 24 Jul 2019 20:26:06 +0300
Subject: [PATCH] staging: dpaa2-mac: Update interface mode array

We used to set PHY_INTERFACE_MODE_XGMII as a placeholder
for interface modes listed by MC but not defined in the
linux kernel. Some of these modes have been added in upstream,
so update the interface mode array to better match actual PHYs.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
---
 drivers/staging/fsl-dpaa2/mac/mac.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -77,11 +77,11 @@ static phy_interface_t dpaa2_mac_iface_m
 	PHY_INTERFACE_MODE_RGMII,	/* DPMAC_ETH_IF_RGMII */
 	PHY_INTERFACE_MODE_SGMII,	/* DPMAC_ETH_IF_SGMII */
 	PHY_INTERFACE_MODE_QSGMII,	/* DPMAC_ETH_IF_QSGMII */
-	PHY_INTERFACE_MODE_XGMII,	/* DPMAC_ETH_IF_XAUI */
+	PHY_INTERFACE_MODE_XAUI,	/* DPMAC_ETH_IF_XAUI */
 	PHY_INTERFACE_MODE_XGMII,	/* DPMAC_ETH_IF_XFI */
 	PHY_INTERFACE_MODE_XGMII,        /* DPMAC_ETH_IF_CAUI */
-	PHY_INTERFACE_MODE_XGMII,       /* DPMAC_ETH_IF_1000BASEX */
-	PHY_INTERFACE_MODE_XGMII,       /* DPMAC_ETH_IF_USXGMII */
+	PHY_INTERFACE_MODE_1000BASEX,   /* DPMAC_ETH_IF_1000BASEX */
+	PHY_INTERFACE_MODE_USXGMII,     /* DPMAC_ETH_IF_USXGMII */
 };
 
 static int cmp_dpmac_ver(struct dpaa2_mac_priv *priv,
